--- origsrc/qsynth-0.3.6/src/CMakeLists.txt	2010-10-31 13:44:59.000000000 -0500
+++ src/qsynth-0.3.6/src/CMakeLists.txt	2011-10-23 21:32:51.584331200 -0500
@@ -126,7 +126,7 @@ if (UNIX AND NOT APPLE)
     install ( TARGETS qsynth
               RUNTIME DESTINATION bin )
     install ( FILES ${QM_FILES}
-              DESTINATION share/locale )
+              DESTINATION share/qsynth )
     install ( FILES ${CMAKE_CURRENT_BINARY_DIR}/qsynth.desktop
               DESTINATION share/applications )
     install ( FILES images/qsynth.png
@@ -137,5 +137,5 @@ if (WIN32)
     install ( TARGETS qsynth
               RUNTIME DESTINATION . )
     install ( FILES ${QM_FILES}
-              DESTINATION share/locale )
+              DESTINATION share/qsynth )
 endif (WIN32)
--- origsrc/qsynth-0.3.7/src/qsynth.cpp	2013-02-14 08:52:31.409468000 -0600
+++ src/qsynth-0.3.7/src/qsynth.cpp	2013-08-01 18:52:17.878040100 -0500
@@ -55,7 +55,7 @@ const WindowFlags CustomizeWindowHint
 #if defined(LOCALEDIR)
 #define CONFIG_LOCALEDIR CONFIG_QUOTED(LOCALEDIR)
 #else
-#define CONFIG_LOCALEDIR CONFIG_DATADIR "/locale"
+#define CONFIG_LOCALEDIR CONFIG_DATADIR "/qsynth"
 #endif
 
 
