--- origsrc/qsynth-0.3.6/CMakeLists.txt	2011-04-02 07:39:39.000000000 -0500
+++ src/qsynth-0.3.6/CMakeLists.txt	2011-10-23 21:36:51.714823900 -0500
@@ -33,6 +33,10 @@ include (CheckIncludeFiles)
 include (CheckFunctionExists)
 include (CheckLibraryExists)
 
+if (Q_WS_X11)
+  find_package (X11 REQUIRED)
+endif (Q_WS_X11)
+
 # Checks for libraries.
 if (WIN32)
   check_function_exists ( lroundf CONFIG_ROUND )
--- origsrc/qsynth-0.3.6/src/CMakeLists.txt	2010-10-31 13:44:59.000000000 -0500
+++ src/qsynth-0.3.6/src/CMakeLists.txt	2011-10-23 21:32:51.584331200 -0500
@@ -110,6 +110,7 @@ target_link_libraries ( qsynth
     ${QT_LIBRARIES}
     ${MATH_LIBRARY}
     ${FLUIDSYNTH_LIBRARY}
+    ${X11_X11_LIB}
 )
 
 set ( TRANSLATIONS
